.gitignore

```diff
@@ -104,3 +104,7 @@ dist
 .tern-port
 build/
 api/package-lock.json
```

Comment:
I believe we also want to ignore the frontend `package-lock`, which I think may be committed in this PR.

- [ ] Ignore?

Reply:
fixed